usr/lib/byobu/logo

```diff
@@ -115,6 +115,10 @@ __logo() {
 			logo="OWrt"
 			$MARKUP && printf "$(color b colour66 W)%s$(color -)" "$logo" || printf "$logo"
 		;;
+		*raspbian*)
+			logo=" @ "
+			$MARKUP && printf "$(color colour125 colour15)%s$(color -)" "$logo" || printf "$logo"
```

Comment:
funny that you use two different spelling of color vs colour

Reply:
Raspberry Pi Foundation is UK-based so I'm guessing colour is spelled the British/Commonwealth way by the Raspbian folks, and without looking my guess is that colour is inherited from that (though I'm guessing, haven't reviewed the change; as long as it works IDC (besides as a Canadian preferring the British spelling, but not enough to die on that hill).

Reply:
I submitted this PR and I'm not affiliated with the Raspberry Pi Foundation. I'm also American.

For my PR, `colour` is used to simply follow convention within the Byobu codebase. It wasn't a decision I made. The color variables for various other distros supported are using the same spelling. Why it started that way in the first place, that's a different question, perhaps for Dustin.

Reply:
Partially right about inheriting the spelling, but not from Pi environment but Byobu.  I hadn't noticed that...

Reply:
It's just because it's using terminology from two different code bases.  I'm the maintainter of Byobu (and American), and in my Byobu code, I used "color".  However, I'm calling out to a function of tmux, where its function is named "colour*".  In any case, the patch is correct, and included upstream.  Thanks!